### DISCUSSION TITLE
Welcome to Celo Composer Discussions!

### DISCUSSION THREAD

**Discussion by joenyzio** [celo-org/celo-composer#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SCHMOTT22** [comment]
I am interested in creating a project with a new platform such as Cleo just to see if I can do it

**Reply by SCHMOTT22** [reply]
[]()

**Comment by DevBigEazi** [comment]
The celo composer cli is creating react app directly, meanwhile I am interested in building with React-Native
